tests/test_admin_check_status_co.py

```diff
@@ -0,0 +1,11 @@
+from conftest import app, prep_db
+from utilities.db import prepare_db
+
+
+# def test_admin_check_status_co(app, prep_db):
+def test_admin_check_status_co(app, make_screen):
+    # prepare_db()
+    app.header.select_option('Log in')
+    app.sign_in_page.login('ADMIN')
+
+    assert app.companies_page.view_status_of_co()
```

Comment:
Method name should be something like is_company_status_visible

Reply:
Done